extensions/amp-story/1.0/amp-story-interactive.js

```diff
@@ -698,6 +728,7 @@ export class AmpStoryInteractive extends AMP.BaseElement {
 
     if (this.optionsData_) {
       this.rootEl_.classList.add('i-amphtml-story-interactive-has-data');
+      this.updateOptionPercentages_(this.optionsData_);
```

Comment:
Why this change?

Reply:
Saw that everywhere that updateToPostSelectionState was called, also updateOptionPercentages was being called. That's because updating the option percentages was part of the post-selection state for components, so I just moved the calls to this method to simplify the code.